front-end/src/app/reports/transactions/transaction-list/transaction-loans-and-debts/transaction-loans-and-debts.component.html

```diff
@@ -61,7 +61,7 @@ <h5 class="gandhi-bold size-24">Loans and Debts</h5>
           {{ item[item.transactionType.templateMap.date] | fecDate }}
         </td>
         <td>
-          {{ item['amount'] | currency }}
+          {{ item['loans_debts_transaction_table_amount'] | currency }}
```

Comment:
I'm wondering if this identifier should be so specific. If it comes up down the road that this value will be used elsewhere outside the transaction table as well. What about "debt_incurred_amount" since that is the origin of the amount?

Reply:
Yes, I think that is a great idea - pushed changes with this.  My only concern is that sometimes it could be the debt incurred amount and sometimes it could be another amount (in which case we won't care about it), but I still think this makes the most sense.

Reply:
for clarity, the corresponding api changes (notice it defaults to amount):

```
debt_incurred_amount=Case(
                    When(
                        schedule_d__isnull=False,
                        debt__isnull=False,
                        then=F("debt__schedule_d__incurred_amount"),
                    ),
                    default=F("amount"),
                ),
```